Makefile

```diff
@@ -98,7 +98,7 @@ test: test-journald ## Run unit tests
 
 .PHONY:integration-test
 integration-test: ## Run integration tests
-	$(RUST_COMMAND) "--env LOGDNA_INGESTION_KEY=$(LOGDNA_INGESTION_KEY) --env LOGDNA_HOST=$(LOGDNA_HOST) --env RUST_BACKTRACE=full" "cargo test --manifest-path bin/Cargo.toml --features integration_tests -- --nocapture"
+	$(RUST_COMMAND) "--env LOGDNA_INGESTION_KEY=$(LOGDNA_INGESTION_KEY) --env LOGDNA_HOST=$(LOGDNA_HOST) --env RUST_BACKTRACE=full" "cargo test --manifest-path bin/Cargo.toml --features integration_tests -- --nocapture --test-threads=1"
```

Comment:
Hmm, what errors were you running in to running the tests in parallel?

Reply:
No errors, it's just that starting multiple agents in parallel and trying to debug/see output makes everything harder.

Do you prefer to continue having integration tests in parallel?

Reply:
If we can run them in parallel I'd prefer to, Maybe we should paramaterise it in the makefile?

Reply:
will do.